### ISSUE TITLE
Component: Menubar "Duplicate id aria" a11y

### ISSUE THREAD

**Issue by Draganlazarev90** [primefaces/primeng#14972]
### Describe the bug

![image](https://github.com/primefaces/primeng/assets/23295567/ccbd7f17-3d5a-445f-902b-6ce61fbee49d)
WCAG violation "Document has multiple elements referenced with ARIA with the same id attribute: pn_id_4"

### Environment

primeng 17.9

### Angular version

17.0.5

### PrimeNG version

17.9

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0
